server/src/main/java/org/elasticsearch/cluster/metadata/MetaDataIndexStateService.java

```diff
@@ -554,4 +573,8 @@ public static ClusterBlock createIndexClosingBlock() {
             EnumSet.of(ClusterBlockLevel.WRITE));
     }
 
+    public static boolean isIndexVerifiedBeforeClosed(final IndexMetaData indexMetaData) {
+        return indexMetaData.getState() == IndexMetaData.State.CLOSE
+            && indexMetaData.getSettings().getAsBoolean(VERIFIED_BEFORE_CLOSE_SETTING.getKey(), false);
```

Comment:
Use `VERIFIED_BEFORE_CLOSE_SETTING.get(indexMetaData.getSettings())` so that the default is not defined in two places?

Reply:
Ok